### ISSUE TITLE
Provide an Ant build file

### ISSUE THREAD

**Issue by xxv** [mttkay/droid-fu#31]
Perhaps this is just a personal preference, but it would be very nice if this project didn't require Maven in order to build it. This project is my first attempt to use Maven and it's already leading to hours of frustration and no jars. Using Maven for Android stuff is uncommon, which then leaves a bunch of non-Maven-familiar developers banging their heads for hours trying to get their first build going. 

From what I've seen, most Android projects/developers use Eclipse, some use Ant and a few use Maven. Following this distribution could help more Android developers use this library.

**Comment by mttkay** [comment]
fair enough. personally I neither know nor like Ant particularly well, but perhaps someone else can provide a working build.xml as a patch

just curious, what are you struggling with specifically? I recently updated the README with what I hoped would be a detailed step-by-step walkthrough how to build and deploy the JAR, but apparently I failed :-P

**Comment by xxv** [comment]
Well, to start, I was really not excited about all the stuff Maven had to download in order to do basic tasks. I run Debian, so I thought it'd be able to just use the plugins that I installed. There seems to be some complexity about how that all works together and there were a couple random errors during this process that stopped the process for completing. I don't even think Maven got to the point where it could start compiling droid-fu or any of its immediate dependencies. 

As far as ant: if I end up using Droid-Fu, I'll look into making an ant build. At the moment, unfortunately it's looking like it doesn't actually do the image caching that thought it did.

As an aside: after giving up on Maven, I was able to make it go in Eclipse with a tinkering of the build path. I had to disable some of the tests, though.
